### PR TITLE
Deemphasize 'halt' subcommand

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/poweroff.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
   <refsect1>
     <title>Description</title>
 
-    <para><command>halt</command>, <command>poweroff</command>, <command>reboot</command> may be used to
+    <para><command>poweroff</command>, <command>halt</command>, <command>reboot</command> may be used to
```
Comment (poettering) [inline man/poweroff.xml]: might as well move "halt" last?

File: man/poweroff.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@
...
     halt, power-off, or reboot the machine. All three commands take the same options.</para>
```
Comment (poettering) [inline man/poweroff.xml]: change order here too

File: man/systemd-poweroff.service.xml [hunk 3]
```diff
@@ -36,11 +36,11 @@
...
     <title>Description</title>
 
-    <para><filename>systemd-halt.service</filename> is a system
-    service that is pulled in by <filename>halt.target</filename> and
-    is responsible for the actual system halt. Similarly,
-    <filename>systemd-poweroff.service</filename> is pulled in by
-    <filename>poweroff.target</filename>,
+    <para><filename>systemd-poweroff.service</filename> is a system
+    service that is pulled in by <filename>poweroff.target</filename> and
+    is responsible for the actual system poweroff. Similarly,
```
Comment (poettering) [inline man/systemd-poweroff.service.xml]: poweroff → power-off?

Comment (poettering) [reply man/systemd-poweroff.service.xml]: or even "power-off operation"?

**Review by bluca — APPROVED** [review]

**Pull request merged**